packages/ripple-nuxt-tide/lib/core/config-loader.js

```diff
@@ -210,6 +212,6 @@ export const build = (tideConfig, _this) => {
   // Build custom modules
   buildCustomModules(tideConfig, _this)
   if (process.env.TIDE_DEBUG) {
-    console.info('Tide configuration:', tideConfig)
+    logger.log('debug', 'Tide configuration: %O', tideConfig)
```

Comment:
What's the difference between %s and %0?

Reply:
Winston log format is using node `util.format` You can check the details here: https://nodejs.org/api/util.html#util_util_format_format_args.

Reply:
Cheers